src/redux/wishlist/wishlist.sagas.js

```diff
@@ -0,0 +1,40 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
+
+import { setWishlist } from './wishlist.actions';
+import {
+  GET_WISHLIST,
+  ADD_ITEM_TO_WISHLIST,
+  REMOVE_ITEM_FROM_WISHLIST
+} from './wishlist.types';
+
+import {
+  getFromLocalStorage,
+  setToLocalStorage
+} from '../../services/local-storage.service';
+
+function* handleWishlistLoad() {
+  const wishlist = yield call(getFromLocalStorage, 'wishlist');
```

Comment:
There is no need to use `yield call` here. `getFromLocalStorage` is a sync function